staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go

```diff
@@ -240,18 +241,23 @@ func newJSONPathParser(jsonPathExpression string) (*jsonpath.JSONPath, error) {
 	return j, nil
 }
 
-// processJSONPathInput will parses the user's JSONPath input and process the string
-func processJSONPathInput(jsonPathExpression, jsonPathCond string) (string, string, error) {
-	relaxedJSONPathExp, err := cmdget.RelaxedJSONPathExpression(jsonPathExpression)
+// processJSONPathInput will parses the user's JSONPath input containing JSON expression and, optionally, JSON value for matching condition and process it
+func processJSONPathInput(jsonPathInput []string) (string, string, error) {
+	if numOfArgs := len(jsonPathInput); numOfArgs < 1 || numOfArgs > 2 {
+		return "", "", fmt.Errorf("jsonpath wait format must be --for=jsonpath='{.status.readyReplicas}'=3 or --for=jsonpath='{.status.readyReplicas}'")
+	}
+	relaxedJSONPathExp, err := cmdget.RelaxedJSONPathExpression(jsonPathInput[0])
 	if err != nil {
 		return "", "", err
 	}
-	if jsonPathCond == "" {
-		return "", "", errors.New("jsonpath wait condition cannot be empty")
+	if len(jsonPathInput) > 1 {
+		jsonPathValue := jsonPathInput[1]
```

Comment:
Could you please add a comment above this to show that an example of what `jsonPathInput[1]` can be?

Reply:
I have changed the wording of the error message. Instead of
`jsonpath wait condition cannot be empty`
it says
`jsonpath wait has to have a value after equal sign, like --for=jsonpath='{.status.readyReplicas}'=3`

I am not sure what value the comment with an example of `jsonPathInput[1]` value brings here. Any non-empty string will do (at parsing phase). I looked at "[old code][1]" and it does the same thing: storing a token as a string. It just stores it to a variable with a different name.
Would you agree that both variable names are self-descriptive regarding expectations for the value of `jsonPathInput[1]`?

[1]: https://github.com/kubernetes/kubernetes/blob/8ffbbe455b30d030a8c99857ad85282f6aec95e6/staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go#L213